src/tlo/methods/wasting.py

```diff
@@ -1,41 +1,1290 @@
+            Types.REAL, 'coverage of outpatient therapeutic care for SAM in health centres'),
+        'coverage_inpatient_care': Parameter(
+            Types.REAL, 'coverage of inpatient care for complicated SAM in hospitals'),
+        'prob_mam_after_care': Parameter(
+            Types.REAL, 'probability of returning to MAM after seeking care'),
+        'prob_death_after_care': Parameter(
+            Types.REAL, 'probability of dying after seeking care'),
+        # recovery due to treatment/interventions
+        'recovery_rate_with_standard_RUTF': Parameter(
+            Types.REAL, 'probability of recovery from wasting following treatment with standard RUTF'),
+        'recovery_rate_with_soy_RUSF': Parameter(
+            Types.REAL, 'probability of recovery from wasting following treatment with soy RUSF'),
+        'recovery_rate_with_CSB++': Parameter(
+            Types.REAL, 'probability of recovery from wasting following treatment with CSB++'),
+        'recovery_rate_with_inpatient_care': Parameter(
+            Types.REAL, 'probability of recovery from wasting following treatment with inpatient care'),
+        'tx_length_weeks_SuppFeedingMAM': Parameter(
+            Types.REAL, 'number of weeks the patient receives treatment in the Supplementary Feeding '
+                        'Programme for MAM before being discharged'),
+        'tx_length_weeks_OutpatientSAM': Parameter(
+            Types.REAL, 'number of weeks the patient receives treatment in the Outpatient Therapeutic '
+                        'Programme for SAM before being discharged if they do not die beforehand'),
+        'tx_length_weeks_InpatientSAM': Parameter(
+            Types.REAL, 'number of weeks the patient receives treatment in the Inpatient Care for complicated'
+                        ' SAM before being discharged if they do not die beforehand'),
+    }
 
     PROPERTIES = {
-        'un_clinical_acute_malnutrition': Property(Types.CATEGORICAL,
-                                                   'temporary property', categories=['MAM', 'SAM', 'well']),
-        'un_ever_wasted': Property(Types.BOOL, 'temporary property')
+        # Properties related to wasting
+        'un_ever_wasted': Property(Types.BOOL, 'ever had an episode of wasting (WHZ < -2)'),
+        'un_WHZ_category': Property(Types.CATEGORICAL, 'weight-for-height Z-score category',
+                                    categories=['WHZ<-3', '-3<=WHZ<-2', 'WHZ>=-2']),
+        'un_last_wasting_date_of_onset': Property(Types.DATE, 'date of onset of last episode of wasting'),
+
+        # Properties related to clinical acute malnutrition
+        'un_clinical_acute_malnutrition': Property(Types.CATEGORICAL, 'clinical acute malnutrition state based'
+                                                                      ' on WHZ and/or MUAC and/or oedema',
+                                                   categories=['MAM', 'SAM'] + ['well']),
```

Comment:
@EvaJanouskova is there a particular reason why categories was changed from `categories=['MAM', 'SAM', 'well']`? The way it is written right now is normally used if the list changes dynamically which creates an overhead although usually negligible. After going through the code, the list would work if it was still static with `categories=['MAM', 'SAM', 'well']`